idxmgmt/indices.go

```diff
@@ -54,7 +54,7 @@ func eventIdxNames(dateSuffix bool) map[string]string {
 		suffix = apmSuffix
 	}
 	idcs := map[string]string{}
-	for _, k := range []string{"span", "transaction", "error", "metric"} {
+	for _, k := range []string{"span", "transaction", "error", "metric", "profile"} {
```

Comment:
Please also add to the policies https://github.com/elastic/apm-server/blob/master/idxmgmt/ilm/policy.go#L44

Reply:
Done. BTW, I intentionally didn't add the profile event to docs or examples in apm-server.yml, given its experimental status.